LoopKit/DoseMath.swift

```diff
@@ -0,0 +1,576 @@
+//
+//  DoseMath.swift
+//  Naterade
+//
+//  Created by Nathan Racklyeft on 3/8/16.
+//  Copyright © 2016 Nathan Racklyeft. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+
+private enum InsulinCorrection {
```

Comment:
Difficult to remember which files are just copied over vs those that were copies and have new code. Are you able to highlight any new code in such files?

Reply:
Are there new tests that support this new functionality?

Reply:
This is another class just copied over as is.  The private `InsulinCorrection` class does not have any separate unit tests, but the methods that use them do have coverage.